orc-rt/docs/README.txt

```diff
@@ -0,0 +1,13 @@
+ORC-RT Documentation
+====================
+
+The ORC-RT documentation is written using the Sphinx documentation generator. It is
+currently tested with Sphinx 1.1.3.
```

Comment:
Sphinx 1.1.3 is ancient (2012). Is this copy-pasta from elsewhere?

Reply:
This is copy-pasta from various existing CMakeFiles. We should update this prior to merging.